app/src/main/java/me/saket/dank/widgets/IndentedLayout.java

```diff
@@ -10,23 +10,70 @@
 import androidx.annotation.IntRange;
 import androidx.annotation.Nullable;
 import androidx.annotation.Px;
+
 import android.util.AttributeSet;
 import android.util.TypedValue;
 import android.widget.LinearLayout;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import me.saket.dank.R;
 
 public class IndentedLayout extends LinearLayout {
-
-  private static final int DEFAULT_SPACING_PER_DEPTH_DP = 8;
-  private static final int DEFAULT_LINE_WIDTH = 4;
+  private static final int DEFAULT_SPACING_PER_DEPTH_DP = 10;
+  private static final int DEFAULT_LINE_WIDTH = 6;
 
   private int indentationDepth;
   private final int spacePerDepthPx;
   private Paint indentationLinePaint;
-  private Path indentationLinePath;
+  private int indentationLineWidth;
 
   private int originalPaddingStart;
+  private boolean coloredDepth = true; // read for preferences ...
```

Comment:
Thank you for this! No need to observe the preferences since you need to close the submission view to access the settings so it will redraw again with the new setting in place.

Reply:
True, except there is a rare situation where someone could open submission in a split window and settings screen in another. In that case the preference won't take effect in the second window until they reopen it.